models/UsersModel.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs').promises;
+
+const connect = async () => {
+    const usersJSON = await fs.readFile("users.json", "utf-8");
+}
+
+class UsersModel {
+
+    constructor(obj){
+        fs.readFile("users.json", "utf-8").then(usersJSON => {
+            this.object = JSON.parse(usersJSON)
+            this.array = this.object.users;
+        });
+    }
+
+    async add(email, password){
+        this.array.push({email: email, password: password});
+        await this.save();
+        return {email}
+    }
+
+    isExists(email){
+        if(this.array.some(user => user.email === email)) return true;
```

Comment:
As before, the function could be renamed to `exists` and rewritten into `return this.array.some(user => user.email === email));`